SonarQube.Client/Api/V8_6/GetHotspotRequest.cs

```diff
@@ -38,20 +36,29 @@ public class GetHotspotRequest : RequestBase<SonarQubeHotspot>, IGetHotspotReque
         protected override SonarQubeHotspot ParseResponse(string response)
         {
             var serverResponse = JObject.Parse(response).ToObject<GetHotspotResponse>();
+
+            var rule = new SonarQubeHotspotRule(
+                serverResponse.Rule.Key,
+                serverResponse.Rule.Name,
+                serverResponse.Rule.SecurityCategory,
+                serverResponse.Rule.VulnerabilityProbability,
+                serverResponse.Rule.RiskDescription,
+                serverResponse.Rule.VulnerabilityDescription,
+                serverResponse.Rule.FixRecommendations
+            );
```

Comment:
I moved the rule definition data into a separate to make it easier to work with.

Reply:
Good call, it started to be a lot